openerp-client/src/main/java/org/bahmni/openerp/web/request/builder/RequestBuilder.java

```diff
@@ -59,4 +61,38 @@ private static String buildRequest(String templateName, HashMap<String, Object>
             throw new OpenERPException(e);
         }
     }
+    public static String buildNewJSONObject(OpenERPRequest openERPRequest, String id) {
```

Comment:
Plz add a unit test for this functionality.

Reply:
Sure, will add it

Reply:
@gsluthra Added test case, Pls have a look on it